txsni/snimap.py

```diff
@@ -67,8 +67,8 @@ def __getattr__(self, attr):
     def __setattr__(self, attr, val):
         if attr in ('_obj', '_factory'):
             self.__dict__[attr] = val
-
-        return setattr(self._obj, attr, val)
+        else:
+            setattr(self._obj, attr, val)
```

Comment:
`_ContextProxy` has an identical `__setattr__` and probably also needs to be updated? https://github.com/glyph/txsni/blob/master/txsni/snimap.py#L106-L110